### ISSUE TITLE
Feature Request: Segment Domain Proxy

### ISSUE THREAD

**Issue by nason** [DavidWells/analytics#92]
Thanks for the great library! We're using `@analytics/segment` and would like to load their script from our custom domain proxy (see https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/custom-proxy/ for details on that)

If the `cdn.segment.com` on this line was configurable, we'd be set!

https://github.com/DavidWells/analytics/blob/6ef1aec8d87979327cadc74dc1f979f678d59488/packages/analytics-plugin-segment/src/browser.js#L139

Do you have any thoughts on adding this feature? I'd be happy to open a PR if you're open to it

**Comment by nason** [comment]
I had some time today so I went ahead with the PR. I'm very open to feedback, so let me know if you have any thoughts or concerns!

**Comment by DavidWells** [comment]
This is available now in `@analytics/segment@0.4.3` via `customScriptSrc`

Thanks for the PR!

Added docs on how to use on the site here: https://getanalytics.io/plugins/segment/#loading-script-from-custom-proxy

```js
import Analytics from 'analytics'
import segmentPlugin from '@analytics/segment'

const analytics = Analytics({
  app: 'awesome-app',
  plugins: [
    segmentPlugin({
      writeKey: '123-xyz',
      // Load segment's analytics.js from somewhere else
      customScriptSrc: 'https://yoursite.com/my-custom-loader.js'
    })
  ]
})
```
